### ISSUE TITLE
Can't press enter while the fig window is open

### ISSUE THREAD

**Issue by MarkusAbtion** [withfig/fig#1973]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.0 (21G115)

### Expected behaviour

I expected the command in my terminal to run, regardless of what happens in the fig window.

### Actual behaviour

I can't run a command until I've ESC'd out of the window

### Environment

```yaml
fig-details:
  - 2.7.2
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 10
  - mem: 16.00 GB
os-info:
  - macOS 12.6.0 (21G115)
environment:
  - shell: /bin/zsh
  - terminal: vscode
  - cwd: 
  - exe-path: /.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 87a34bb4-3f63-47f4-9e40-c8829fc54bfb
    - FIG_HOSTNAME: HUAWEI_P10-bd76470d46f906.localdomain
    - FIG_PID: 95402
    - FIG_TERM: 2.7.2
    - PATH: [REDACTED]
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PARENT: 87a34bb4-3f63-47f4-9e40-c8829fc54bfb
    - FIG_SET_PARENT: 87a34bb4-3f63-47f4-9e40-c8829fc54bfb
```

**Comment by nunorafaelmacedo** [comment]
Same problem here.

For me the ENTER key isn't recognised on Fig even, if I assign ENTER in to "Insert selection" it doesn't even work, only with TAB for example.

**Comment by karlmarxlopez** [comment]
Just to add more context to this, ENTER does work if you move the cursor to the left of the last character.

**Comment by mschrage** [comment]
Hey @karlmarxlopez - I believe this issue has been addressed on the latest beta (`2.7.6-beta.2`). 

Can you run `fig settings app.beta true` and confirm that everything is working as expected?
